docs/mpdev-references.md

```diff
@@ -162,12 +161,20 @@ APPLICATION_NAMESPACE=default
 # no typos in the names. This should succeed.
 kubectl get application/$APPLICATION_INSTANCE -n $APPLICATION_NAMESPACE
 
+# Use the following for schema v2.
```

Comment:
Maybe include how to tell if the application uses schema v1 vs v2?

Reply:
Done.